ssl/record/methods/dtls_meth.c

```diff
@@ -170,8 +170,7 @@ static int dtls_process_record(OSSL_RECORD_LAYER *rl, DTLS_BITMAP *bitmap)
         mac = rr->data + rr->length;
         i = rl->funcs->mac(rl, rr, md, 0 /* not send */);
         if (i == 0 || CRYPTO_memcmp(md, mac, (size_t)mac_size) != 0) {
-            RLAYERfatal(rl, SSL_AD_BAD_RECORD_MAC,
-                        SSL_R_DECRYPTION_FAILED_OR_BAD_RECORD_MAC);
+            /* Silently drop the packet in case of invalid MAC. */
```

Comment:
I'd like a reference in the comment as well:
 "As per RFC 9147 4.5.2: Silently drop the packet in case of invalid MAC."